internal/jobs/scheduler.go

```diff
@@ -449,6 +449,9 @@ func (s *Scheduler) verify(jobConfiguration *JobConfiguration) error {
 	if len(jobConfiguration.Id) <= 0 {
 		return errors.New("job configuration needs an id")
 	}
+	if len(jobConfiguration.Title) <= 0 {
+		return errors.New("job configuration needs a title")
```

Comment:
just a thought, maybe we should consider checking for duplicate titles as well. Not sure it this is the correct place - "addJob" is a good place to check for duplicates

Reply:
I agree. We currently do that in the deploy tool, but it would be nice to detect it here as well. Not everyone will use the deploy tool at all times, so we should just check it internally regardless. I'll have a look at it.

Reply:
@rompetroll I've added a check for unique title. Please have a look.